daemon/algod/api/algod.oas2.json

```diff
@@ -4244,6 +4277,13 @@
               "$ref": "#/definitions/SimulateTransactionGroupResult"
             }
           },
+          "lift-log-limits": {
```

Comment:
I don't think this needs to be reported here, since `eval-changes` will already indicate limits are different

Reply:
see [refactor Simulate interface, remove unnecessary codec struct, remove …](https://github.com/algorand/go-algorand/pull/5247/commits/61d8b7e19f08848a812d01c8933950740c97b60d)